package.json

```diff
@@ -95,9 +95,6 @@
   },
   "devDependencies": {
     "@build-tracker/cli": "^1.0.0-beta.15",
-    "@firebase/app-types": "0.3.1",
-    "@firebase/auth-types": "0.3.2",
-    "@firebase/util": "0.2.1",
```

Comment:
I believe these were a merge error. Removed when firebase was updated for Viewer in #13115, but added back in #12771. Could not figure out what part of the rollup process they were for, until I remembered that was just a long-lived PR and those lines probably just snuck back in. cc @connorjclark just in case